shareMethod.sh

```diff
@@ -9,9 +9,9 @@ execWithTimeout() {
         echo "execWithTimeout() Command not specified"
         exit 2
     fi
-    local cmd="/bin/sh -c \"$1\""
+    export cmd="/bin/sh -c \"$1\""
```

Comment:
@PengTian0  @panpan0000  the same problem with https://stackoverflow.com/questions/21928226/how-to-pass-shell-script-variable-into-expect-script  this link could help you understand the background